components/org.wso2.carbon.siddhi.editor.core/src/main/java/org/wso2/carbon/siddhi/editor/core/internal/EditorMicroservice.java

```diff
@@ -874,6 +876,39 @@ public Response getCodeView(String encodedEventFlowJson) {
         }
     }
 
+    @GET
+    @Path("/siddhi-apps/{appName}/streams/{streamName}/event/{type}")
+    @Produces({"text/plain"})
+    public Response getDefaultSampleStreamEvent(@PathParam("appName") String appName,
+                                                @PathParam("streamName") String streamName,
+                                                @PathParam("type") String eventType)
+            throws NotFoundException {
+        SiddhiAppRuntime siddhiAppRuntime = EditorDataHolder.getSiddhiManager().getSiddhiAppRuntime(appName);
+        if (siddhiAppRuntime == null) {
+            String jsonString = new Gson().toJson("There is no Siddhi App exist with provided name : " + appName);
```

Comment:
since this is a error case can we have a format like error:errorMsg and send it to client side ?

Reply:
@dnwick yes, this 500 error is caught in client side and shown in the notification area

Reply:
@ramindu90 ya but you are constructing a json here right. Can we have a consistant format like Key is "error" and value is what you have put above ? like "error":"There is no Siddhi App exist with provided name : " + appName" ?
So from client side you only need to get the key as error in all error cases ?

Reply:
@dnwick the error will be shown as follows.

<img width="1440" alt="screen shot 2018-11-29 at 12 48 43 pm" src="https://user-images.githubusercontent.com/10056966/49205349-c47ddf80-f3d4-11e8-8043-366e0e1974c0.png">

Reply:
@dnwick errors will be shown as follows
<img width="1434" alt="screen shot 2018-11-29 at 12 48 43 pm" src="https://user-images.githubusercontent.com/10056966/49205432-f98a3200-f3d4-11e8-9e7a-4525cde86f06.png">

Reply:
@ramindu90 lets change the error message format as discussed